CONTRIBUTING.md

```diff
@@ -1,76 +0,0 @@
-### Contributing to ECMAScript
```

Comment:
i think CONTRIBUTING.md here should stay in this repo, since it's 262-specific.

Reply:
hmmm... it's referenced elsewhere, like https://github.com/tc39/how-we-work#tc39---how-we-work

should we instead create a new more general `CONTRIBUTING.md` for `.github`?